proxy/proxy.go

```diff
@@ -1241,8 +1244,10 @@ func (p *Proxy) errorResponse(ctx *context, err error) {
 		code = p.defaultHTTPStatus
 	case ok && perr.err == errRatelimit:
 		code = perr.code
+	case code == 499:
+		p.log.Errorf("client canceled after %s, route %s with backend %s %s, status code %d: %v", time.Since(ctx.startServe), id, backendType, backend, code, err)
```

Comment:
he first parameter should be %v because we feed it with a time.Duration value

Reply:
Nope, I want to show the string representation, see example: https://play.golang.org/p/ZQW0Pl-UnlA

Reply:
i updated the example a bit: https://play.golang.org/p/ZBwdgnJLmZh
plus, here's the doc, seek for %s: https://godoc.org/fmt
;)

Reply:
Ah, so in this case, the output will still be the same:
```
2009-11-10 23:00:00 +0000 UTC m=+0.000000001
2009-11-10 23:05:41 +0000 UTC m=+341.000000001
5m41s
5m41s
```

but it's good practice to always use `%v` because that way, it will default to the correct type depending on the input?

Reply:
indeed, %v is a safe default. But it's a best practice to use %s %d or similar well known cases, where we know the matched input. There are various reasons, e.g. static checking can detect potential mistakes. It's worth having a peek sometimes in the stdlib implementation for examples about the intended usage.

Reply:
Changed.